commcid.go

```diff
@@ -138,12 +301,44 @@ func validateFilecoinCidSegments(mc FilMultiCodec, mh FilMultiHash, commX []byte
 	return nil
 }
 
+// ConvertDataCommitmentV1V1CIDtoPieceMhCID takes a v1 piece CID and the CommP tree height and produces a
+// piece multihash CID
+//
+// The helpers UnpaddedSizeToV1TreeHeight and Fr32PaddedSizeToV1TreeHeight may help in computing tree height
+func ConvertDataCommitmentV1V1CIDtoPieceMhCID(v1PieceCid cid.Cid, unpaddedDataSize uint64) (cid.Cid, error) {
```

Comment:
Too many `V1`s ? Also... the helper seems overly trivial, perhaps drop it? If kept: simpler name below:

```suggestion
func PieceCidV2FromV1(v1PieceCid cid.Cid, unpaddedDataSize uint64) (cid.Cid, error) {
```